service/pom.xml

```diff
@@ -385,7 +385,6 @@ under the License.
                 <configuration>
                     <appendAssemblyId>false</appendAssemblyId>
                     <skipAssembly>false</skipAssembly>
-                    <outputDirectory>../target</outputDirectory>
```

Comment:
Why move it?  I think it's appropriate to put the deployment package at root directory.

Reply:
The service-${version}.tar.gz location is not align with the Documention in https://github.com/apache/griffin/blob/master/griffin-doc/deploy/deploy-guide.md, or we can update the documentation to remove the confuse.  

>>>It's easy to build Griffin, just run maven command mvn clean install. Successfully building, you can get service-${version}.tar.gz and measure-${version}.jar from target folder in **service** and **measure** module.

Reply:
> The service-${version}.tar.gz location is not align with the Documention in https://github.com/apache/griffin/blob/master/griffin-doc/deploy/deploy-guide.md, or we can update the documentation to remove the confuse.
> 
> > > > It's easy to build Griffin, just run maven command mvn clean install. Successfully building, you can get service-${version}.tar.gz and measure-${version}.jar from target folder in **service** and **measure** module.

You're right. I think it's better to update the doc, and the following commands should also be modified at the same time.

```bash
cd $GRIFFIN_INSTALL_DIR
tar -zxvf service-${version}.tar.gz
```

Reply:
@lipzhu please update the docs instead